addons/website_crm_score/static/src/js/set_score.js

```diff
@@ -25,10 +25,10 @@
         },
         is_tracked: function(val) {
             var obj = website.seo.Configurator.prototype.getMainObject();
-            if (!obj) {
-                return $.Deferred().reject();
-            } else {
+            if (obj && obj.model == 'ir.ui.view') {
```

Comment:
@jem-odoo 
In website_crm_score, one field add *track* in ir.ui.view, but in set_score.js when we promote specific blog its takes *blog.post* not ir.ui.view. Same thing can happen with any other page.
Because of this runbot shows warning. 
So did this change.
Thanks.